### ISSUE TITLE
[Bug] Web-1 & Swagger-UI-1 Builds Fail

### ISSUE THREAD

**Issue by anthonyjdella** [TBD54566975/ssi-service#276]
**Describe the bug**
When running `docker-compose up --build` or `docker-compose up` inside `/build`, my web-1 and swagger-ui-1 builds are not running. They fail and are exited.

```
2023-02-03 22:11:21 {"level":"info","msg":"Starting up...","time":"2023-02-04T06:11:21Z"}
2023-02-03 22:11:21 {"level":"fatal","msg":"could not instantiate config: load toml config: could not load config: config/config.toml: open config/config.toml: operation not permitted","time":"2023-02-04T06:11:21Z"}
```

**To Reproduce**
*Steps to reproduce the behavior:*
1. Installed Docker
2. Installed Mage
3. Go to 'build'
4. Run either `docker-compose up --build` or `docker-compose up`

**Expected behavior**
web-1 and swagger-ui-1 should be Running and available on `localhost:8080`
I have also tried switching the port to `8088` and the same issue is happening.

**Supporting Material**
Other builds are passing, but not web-1 and swagger-ui-1
![Screenshot 2023-02-03 at 10 16 54 PM](https://user-images.githubusercontent.com/26190505/216752356-7b97d811-9796-4e94-a51f-032ee2a8e81b.png)
Screenshot shows port `8088` but I also had the same issue with `8080`

**Environment (please complete the following information):**
 - OS: macOS Ventura
 - Version 13.0

**Additional context**

- Already checked if `8080` and `8088` ports are in use, but they are available.
- `config/config.toml` exists and is unchanged from `main`

@mistermoe

**Comment by andresuribe87** [comment]
Thanks for reporting this @anthonyjdella 

I tried to reproduce this today, but wasn't able to. Can you share the output of the `docker-compose up --build` command? Which commit are you trying this from?

**Comment by anthonyjdella** [comment]
@andresuribe87 

Thanks for the reply, using this commit: `91214a4f53a8a632b0d1d3eaa0ec766b9b199abd (HEAD -> main, origin/main, origin/HEAD) Fix docs build (#275)`

Here's the output of the command:
```
docker-compose up --build
[+] Building 20.7s (34/34) FINISHED                                                                                              
 => [build-swagger-ui internal] load build definition from Dockerfile-swagger                                               0.0s
 => => transferring dockerfile: 297B                                                                                        0.0s
 => [build-gui internal] load build definition from Dockerfile-gui                                                          0.0s
 => => transferring dockerfile: 250B                                                                                        0.0s
 => [build-web internal] load build definition from Dockerfile                                                              0.0s
 => => transferring dockerfile: 420B                                                                                        0.0s
 => [build-swagger-ui internal] load .dockerignore                                                                          0.0s
 => => transferring context: 2B                                                                                             0.0s
 => [build-gui internal] load .dockerignore                                                                                 0.0s
 => => transferring context: 2B                                                                                             0.0s
 => [build-web internal] load .dockerignore                                                                                 0.0s
 => => transferring context: 2B                                                                                             0.0s
 => [build-swagger-ui internal] load metadata for docker.io/library/golang:1.17-alpine                                      1.3s
 => [build-web internal] load metadata for docker.io/library/golang:1.19.4-alpine                                           1.4s
 => [build-gui internal] load metadata for docker.io/library/python:3.10-slim-buster                                        1.3s
 => [auth] library/python:pull token for registry-1.docker.io                                                               0.0s
 => [auth] library/golang:pull token for registry-1.docker.io                                                               0.0s
 => [build-swagger-ui 1/6] FROM docker.io/library/golang:1.17-alpine@sha256:99ddec1bbfd6d6bca3f9804c02363daee8c8524dae50df  0.0s
 => [build-swagger-ui internal] load build context                                                                          0.0s
 => => transferring context: 20.71kB                                                                                        0.0s
 => CACHED [build-swagger-ui 2/6] WORKDIR /app                                                                              0.0s
 => CACHED [build-swagger-ui 3/6] COPY go.mod ./                                                                            0.0s
 => CACHED [build-swagger-ui 4/6] COPY go.sum ./                                                                            0.0s
 => CACHED [build-swagger-ui 5/6] RUN go mod download                                                                       0.0s
 => CACHED [build-swagger-ui 6/6] RUN go get -u "github.com/go-swagger/go-swagger/cmd/swagger@v0.29.0"                      0.0s
 => [build-gui] exporting to image                                                                                          1.6s
 => => exporting layers                                                                                                     1.2s
 => => writing image sha256:7162960bd003fdeb4c287aadf93616ddc816353c959928569f96e69300b5bf46                                0.0s
 => => naming to docker.io/library/build-swagger-ui                                                                         0.0s
 => => writing image sha256:fb420ebc10ded221afca028b3809a49db46ccc3866299d02d38470b75b765c73                                0.0s
 => => naming to docker.io/library/build-web                                                                                0.0s
 => => writing image sha256:aef90984198b9fd6edc1fe8e6452712a7d7fe5b628273187b0ef568f6d3bb8cd                                0.0s
 => => naming to docker.io/library/build-gui                                                                                0.0s
 => [build-gui internal] load build context                                                                                 0.0s
 => => transferring context: 4.57kB                                                                                         0.0s
 => [build-gui 1/5] FROM docker.io/library/python:3.10-slim-buster@sha256:ffcccf17ca6bd1ea2f2743fad16e1209d634f1e51a88dbfd  0.0s
 => => resolve docker.io/library/python:3.10-slim-buster@sha256:ffcccf17ca6bd1ea2f2743fad16e1209d634f1e51a88dbfdc589e1dd57  0.0s
 => CACHED [build-gui 2/5] COPY ./gui/requirements.txt /app/requirements.txt                                                0.0s
 => CACHED [build-gui 3/5] WORKDIR /app                                                                                     0.0s
 => [build-gui 4/5] RUN pip install -r requirements.txt                                                                    17.9s
 => [build-web 1/8] FROM docker.io/library/golang:1.19.4-alpine@sha256:86d32cc0dfc04757fd8aeebb86308e6d1e3de60c73cb59e0f99  0.0s
 => [build-web internal] load build context                                                                                 0.2s
 => => transferring context: 3.38MB                                                                                         0.2s
 => CACHED [build-web 2/8] WORKDIR /app                                                                                     0.0s
 => CACHED [build-web 3/8] COPY go.mod ./                                                                                   0.0s
 => CACHED [build-web 4/8] COPY go.sum ./                                                                                   0.0s
 => CACHED [build-web 5/8] RUN go mod download                                                                              0.0s
 => [build-web 6/8] COPY . ./                                                                                               0.1s
 => [build-web 7/8] RUN ls -alt                                                                                             0.2s
 => [build-web 8/8] RUN go build -tags jwx_es256k -o /docker-ssi-service ./cmd                                             15.5s
 => [build-gui 5/5] COPY ./gui /app                                                                                         0.0s
[+] Running 6/5
 ⠿ Container redis-commander     Created                                                                                    0.0s
 ⠿ Container build-jaeger-1      Created                                                                                    0.0s
 ⠿ Container build-redis-1       Created                                                                                    0.0s
 ⠿ Container build-swagger-ui-1  Created                                                                                    0.0s
 ⠿ Container build-web-1         Recreated                                                                                  0.1s
 ⠿ Container build-gui-1         Recreated                                                                                  0.0s
Attaching to build-gui-1, build-jaeger-1, build-redis-1, build-swagger-ui-1, build-web-1, redis-commander
redis-commander     | Creating custom redis-commander config '/redis-commander/config/local-production.json'.
build-redis-1       | 1:C 06 Feb 2023 17:03:26.836 # oO0OoO0OoO0Oo Redis is starting oO0OoO0OoO0Oo
build-redis-1       | 1:C 06 Feb 2023 17:03:26.836 # Redis version=7.0.8, bits=64, commit=00000000, modified=0, pid=1, just started
build-redis-1       | 1:C 06 Feb 2023 17:03:26.836 # Configuration loaded
build-redis-1       | 1:M 06 Feb 2023 17:03:26.837 * monotonic clock: POSIX clock_gettime
build-redis-1       | 1:M 06 Feb 2023 17:03:26.838 * Running mode=standalone, port=6379.
build-redis-1       | 1:M 06 Feb 2023 17:03:26.838 # Server initialized
build-redis-1       | 1:M 06 Feb 2023 17:03:26.840 * Ready to accept connections
redis-commander     | Parsing 1 REDIS_HOSTS into custom redis-commander config '/redis-commander/config/local-production.json'.
build-swagger-ui-1  | open /app/swagger.yaml: operation not permitted
redis-commander     | node ./bin/redis-commander  
build-jaeger-1      | 2023/02/06 17:03:27 maxprocs: Leaving GOMAXPROCS=5: CPU quota undefined
build-swagger-ui-1 exited with code 1
build-jaeger-1      | {"level":"info","ts":1675703007.0343332,"caller":"flags/service.go:119","msg":"Mounting metrics handler on admin server","route":"/metrics"}
build-jaeger-1      | {"level":"info","ts":1675703007.0345566,"caller":"flags/service.go:125","msg":"Mounting expvar handler on admin server","route":"/debug/vars"}
build-jaeger-1      | {"level":"info","ts":1675703007.0350847,"caller":"flags/admin.go:129","msg":"Mounting health check on admin server","route":"/"}
build-jaeger-1      | {"level":"info","ts":1675703007.0351694,"caller":"flags/admin.go:143","msg":"Starting admin HTTP server","http-addr":":14269"}
build-jaeger-1      | {"level":"info","ts":1675703007.0351942,"caller":"flags/admin.go:121","msg":"Admin server started","http.host-port":"[::]:14269","health-status":"unavailable"}
build-jaeger-1      | {"level":"info","ts":1675703007.0370014,"caller":"memory/factory.go:66","msg":"Memory storage initialized","configuration":{"MaxTraces":0}}
build-jaeger-1      | {"level":"info","ts":1675703007.0375354,"caller":"static/strategy_store.go:138","msg":"Loading sampling strategies","filename":"/etc/jaeger/sampling_strategies.json"}
build-jaeger-1      | {"level":"info","ts":1675703007.0558205,"caller":"channelz/funcs.go:340","msg":"[core][Server #1] Server created","system":"grpc","grpc_log":true}
build-jaeger-1      | {"level":"info","ts":1675703007.0560496,"caller":"server/grpc.go:104","msg":"Starting jaeger-collector gRPC server","grpc.host-port":"[::]:14250"}
build-jaeger-1      | {"level":"info","ts":1675703007.0560834,"caller":"server/http.go:56","msg":"Starting jaeger-collector HTTP server","http host-port":":14268"}
build-jaeger-1      | {"level":"info","ts":1675703007.056443,"caller":"channelz/funcs.go:340","msg":"[core][Server #1 ListenSocket #2] ListenSocket created","system":"grpc","grpc_log":true}
build-jaeger-1      | {"level":"info","ts":1675703007.0566518,"caller":"server/zipkin.go:52","msg":"Not listening for Zipkin HTTP traffic, port not configured"}
build-jaeger-1      | {"level":"info","ts":1675703007.0566878,"caller":"grpc/builder.go:73","msg":"Agent requested insecure grpc connection to collector(s)"}
build-jaeger-1      | {"level":"info","ts":1675703007.056771,"caller":"channelz/funcs.go:340","msg":"[core][Channel #3] Channel created","system":"grpc","grpc_log":true}
build-jaeger-1      | {"level":"info","ts":1675703007.056796,"caller":"channelz/funcs.go:340","msg":"[core][Channel #3] original dial target is: \":14250\"","system":"grpc","grpc_log":true}
build-jaeger-1      | {"level":"info","ts":1675703007.0568116,"caller":"channelz/funcs.go:340","msg":"[core][Channel #3] dial target \":14250\" parse failed: parse \":14250\": missing protocol scheme","system":"grpc","grpc_log":true}
build-jaeger-1      | {"level":"info","ts":1675703007.05682,"caller":"channelz/funcs.go:340","msg":"[core][Channel #3] fallback to scheme \"passthrough\"","system":"grpc","grpc_log":true}
build-jaeger-1      | {"level":"info","ts":1675703007.056843,"caller":"channelz/funcs.go:340","msg":"[core][Channel #3] parsed dial target is: {Scheme:passthrough Authority: Endpoint::14250 URL:{Scheme:passthrough Opaque: User: Host: Path:/:14250 RawPath: OmitHost:false ForceQuery:false RawQuery: Fragment: RawFragment:}}","system":"grpc","grpc_log":true}
build-jaeger-1      | {"level":"info","ts":1675703007.0568473,"caller":"channelz/funcs.go:340","msg":"[core][Channel #3] Channel authority set to \"localhost:14250\"","system":"grpc","grpc_log":true}
build-jaeger-1      | {"level":"info","ts":1675703007.0570905,"caller":"channelz/funcs.go:340","msg":"[core][Channel #3] Resolver state updated: {\n  \"Addresses\": [\n    {\n      \"Addr\": \":14250\",\n      \"ServerName\": \"\",\n      \"Attributes\": null,\n      \"BalancerAttributes\": null,\n      \"Type\": 0,\n      \"Metadata\": null\n    }\n  ],\n  \"ServiceConfig\": null,\n  \"Attributes\": null\n} (resolver returned new addresses)","system":"grpc","grpc_log":true}
build-jaeger-1      | {"level":"info","ts":1675703007.057185,"caller":"channelz/funcs.go:340","msg":"[core][Channel #3] Channel switches to new LB policy \"round_robin\"","system":"grpc","grpc_log":true}
build-jaeger-1      | {"level":"info","ts":1675703007.0573657,"caller":"grpclog/component.go:55","msg":"[balancer]base.baseBalancer: got new ClientConn state: {{[{\n  \"Addr\": \":14250\",\n  \"ServerName\": \"\",\n  \"Attributes\": null,\n  \"BalancerAttributes\": null,\n  \"Type\": 0,\n  \"Metadata\": null\n}] <nil> <nil>} <nil>}","system":"grpc","grpc_log":true}
build-jaeger-1      | {"level":"info","ts":1675703007.0574567,"caller":"channelz/funcs.go:340","msg":"[core][Channel #3 SubChannel #4] Subchannel created","system":"grpc","grpc_log":true}
build-jaeger-1      | {"level":"info","ts":1675703007.0574846,"caller":"grpclog/component.go:71","msg":"[roundrobin]roundrobinPicker: Build called with info: {map[]}","system":"grpc","grpc_log":true}
build-jaeger-1      | {"level":"info","ts":1675703007.0574913,"caller":"channelz/funcs.go:340","msg":"[core][Channel #3] Channel Connectivity change to CONNECTING","system":"grpc","grpc_log":true}
build-jaeger-1      | {"level":"info","ts":1675703007.0575392,"caller":"channelz/funcs.go:340","msg":"[core][Channel #3 SubChannel #4] Subchannel Connectivity change to CONNECTING","system":"grpc","grpc_log":true}
build-jaeger-1      | {"level":"info","ts":1675703007.05758,"caller":"channelz/funcs.go:340","msg":"[core][Channel #3 SubChannel #4] Subchannel picks a new address \":14250\" to connect","system":"grpc","grpc_log":true}
build-jaeger-1      | {"level":"info","ts":1675703007.0577025,"caller":"grpc/builder.go:113","msg":"Checking connection to collector"}
build-jaeger-1      | {"level":"info","ts":1675703007.0577388,"caller":"grpc/builder.go:124","msg":"Agent collector connection state change","dialTarget":":14250","status":"CONNECTING"}
build-jaeger-1      | {"level":"info","ts":1675703007.0577755,"caller":"grpclog/component.go:71","msg":"[balancer]base.baseBalancer: handle SubConn state change: 0x4000122af8, CONNECTING","system":"grpc","grpc_log":true}
build-jaeger-1      | {"level":"info","ts":1675703007.058048,"caller":"./main.go:256","msg":"Starting agent"}
build-jaeger-1      | {"level":"info","ts":1675703007.0581634,"caller":"querysvc/query_service.go:135","msg":"Archive storage not created","reason":"archive storage not supported"}
build-jaeger-1      | {"level":"info","ts":1675703007.0582988,"caller":"app/flags.go:136","msg":"Archive storage not initialized"}
build-jaeger-1      | {"level":"info","ts":1675703007.0584424,"caller":"channelz/funcs.go:340","msg":"[core][Channel #3 SubChannel #4] Subchannel Connectivity change to READY","system":"grpc","grpc_log":true}
build-jaeger-1      | {"level":"info","ts":1675703007.0584495,"caller":"channelz/funcs.go:340","msg":"[core][Server #7] Server created","system":"grpc","grpc_log":true}
build-jaeger-1      | {"level":"info","ts":1675703007.0584643,"caller":"grpclog/component.go:71","msg":"[balancer]base.baseBalancer: handle SubConn state change: 0x4000122af8, READY","system":"grpc","grpc_log":true}
build-jaeger-1      | {"level":"info","ts":1675703007.0584896,"caller":"grpclog/component.go:71","msg":"[roundrobin]roundrobinPicker: Build called with info: {map[0x4000122af8:{{\n  \"Addr\": \":14250\",\n  \"ServerName\": \"\",\n  \"Attributes\": null,\n  \"BalancerAttributes\": null,\n  \"Type\": 0,\n  \"Metadata\": null\n}}]}","system":"grpc","grpc_log":true}
build-jaeger-1      | {"level":"info","ts":1675703007.0585055,"caller":"channelz/funcs.go:340","msg":"[core][Channel #3] Channel Connectivity change to READY","system":"grpc","grpc_log":true}
build-jaeger-1      | {"level":"info","ts":1675703007.058519,"caller":"grpc/builder.go:124","msg":"Agent collector connection state change","dialTarget":":14250","status":"READY"}
build-jaeger-1      | {"level":"info","ts":1675703007.0585988,"caller":"channelz/funcs.go:340","msg":"[core][Channel #8] Channel created","system":"grpc","grpc_log":true}
build-jaeger-1      | {"level":"info","ts":1675703007.0586069,"caller":"channelz/funcs.go:340","msg":"[core][Channel #8] original dial target is: \":16685\"","system":"grpc","grpc_log":true}
build-jaeger-1      | {"level":"info","ts":1675703007.0586336,"caller":"channelz/funcs.go:340","msg":"[core][Channel #8] dial target \":16685\" parse failed: parse \":16685\": missing protocol scheme","system":"grpc","grpc_log":true}
build-jaeger-1      | {"level":"info","ts":1675703007.0586388,"caller":"channelz/funcs.go:340","msg":"[core][Channel #8] fallback to scheme \"passthrough\"","system":"grpc","grpc_log":true}
build-jaeger-1      | {"level":"info","ts":1675703007.0586483,"caller":"channelz/funcs.go:340","msg":"[core][Channel #8] parsed dial target is: {Scheme:passthrough Authority: Endpoint::16685 URL:{Scheme:passthrough Opaque: User: Host: Path:/:16685 RawPath: OmitHost:false ForceQuery:false RawQuery: Fragment: RawFragment:}}","system":"grpc","grpc_log":true}
build-jaeger-1      | {"level":"info","ts":1675703007.0586529,"caller":"channelz/funcs.go:340","msg":"[core][Channel #8] Channel authority set to \"localhost:16685\"","system":"grpc","grpc_log":true}
build-jaeger-1      | {"level":"info","ts":1675703007.058662,"caller":"app/agent.go:69","msg":"Starting jaeger-agent HTTP server","http-port":5778}
build-jaeger-1      | {"level":"info","ts":1675703007.0586681,"caller":"channelz/funcs.go:340","msg":"[core][Channel #8] Resolver state updated: {\n  \"Addresses\": [\n    {\n      \"Addr\": \":16685\",\n      \"ServerName\": \"\",\n      \"Attributes\": null,\n      \"BalancerAttributes\": null,\n      \"Type\": 0,\n      \"Metadata\": null\n    }\n  ],\n  \"ServiceConfig\": null,\n  \"Attributes\": null\n} (resolver returned new addresses)","system":"grpc","grpc_log":true}
build-jaeger-1      | {"level":"info","ts":1675703007.058687,"caller":"channelz/funcs.go:340","msg":"[core][Channel #8] Channel switches to new LB policy \"pick_first\"","system":"grpc","grpc_log":true}
build-jaeger-1      | {"level":"info","ts":1675703007.0586994,"caller":"channelz/funcs.go:340","msg":"[core][Channel #8 SubChannel #9] Subchannel created","system":"grpc","grpc_log":true}
build-jaeger-1      | {"level":"info","ts":1675703007.0587397,"caller":"channelz/funcs.go:340","msg":"[core][Channel #8] Channel Connectivity change to CONNECTING","system":"grpc","grpc_log":true}
build-jaeger-1      | {"level":"info","ts":1675703007.058921,"caller":"channelz/funcs.go:340","msg":"[core][Channel #8 SubChannel #9] Subchannel Connectivity change to CONNECTING","system":"grpc","grpc_log":true}
build-jaeger-1      | {"level":"info","ts":1675703007.0590494,"caller":"app/static_handler.go:181","msg":"UI config path not provided, config file will not be watched"}
build-jaeger-1      | {"level":"info","ts":1675703007.0591393,"caller":"app/server.go:218","msg":"Query server started","http_addr":"[::]:16686","grpc_addr":"[::]:16685"}
build-jaeger-1      | {"level":"info","ts":1675703007.0591593,"caller":"healthcheck/handler.go:129","msg":"Health Check state change","status":"ready"}
build-jaeger-1      | {"level":"info","ts":1675703007.0592344,"caller":"app/server.go:301","msg":"Starting GRPC server","port":16685,"addr":":16685"}
build-jaeger-1      | {"level":"info","ts":1675703007.0592446,"caller":"channelz/funcs.go:340","msg":"[core][Server #7 ListenSocket #10] ListenSocket created","system":"grpc","grpc_log":true}
build-jaeger-1      | {"level":"info","ts":1675703007.059266,"caller":"app/server.go:282","msg":"Starting HTTP server","port":16686,"addr":":16686"}
build-jaeger-1      | {"level":"info","ts":1675703007.0592897,"caller":"channelz/funcs.go:340","msg":"[core][Channel #8 SubChannel #9] Subchannel picks a new address \":16685\" to connect","system":"grpc","grpc_log":true}
build-jaeger-1      | {"level":"info","ts":1675703007.0593598,"caller":"grpclog/component.go:71","msg":"[core]pickfirstBalancer: UpdateSubConnState: 0x40001a2750, {CONNECTING <nil>}","system":"grpc","grpc_log":true}
build-jaeger-1      | {"level":"info","ts":1675703007.0597131,"caller":"channelz/funcs.go:340","msg":"[core][Channel #8 SubChannel #9] Subchannel Connectivity change to READY","system":"grpc","grpc_log":true}
build-jaeger-1      | {"level":"info","ts":1675703007.0597374,"caller":"grpclog/component.go:71","msg":"[core]pickfirstBalancer: UpdateSubConnState: 0x40001a2750, {READY <nil>}","system":"grpc","grpc_log":true}
build-jaeger-1      | {"level":"info","ts":1675703007.0597517,"caller":"channelz/funcs.go:340","msg":"[core][Channel #8] Channel Connectivity change to READY","system":"grpc","grpc_log":true}
redis-commander     | Using scan instead of keys
redis-commander     | No Save: false
redis-commander     | listening on 0.0.0.0:8081
redis-commander     | access with browser at http://127.0.0.1:8081
redis-commander     | Redis Connection redis:6379 using Redis DB #0
build-web-1         | {"level":"info","msg":"Starting up...","time":"2023-02-06T17:03:27Z"}
build-web-1         | {"level":"fatal","msg":"could not instantiate config: load toml config: could not load config: config/config.toml: open config/config.toml: operation not permitted","time":"2023-02-06T17:03:27Z"}
build-web-1 exited with code 1
build-gui-1         | 
build-gui-1         | Collecting usage statistics. To deactivate, set browser.gatherUsageStats to False.
build-gui-1         | 
build-gui-1         | 
build-gui-1         |   You can now view your Streamlit app in your browser.
build-gui-1         | 
build-gui-1         |   Network URL: http://172.18.0.5:8501
build-gui-1         |   External URL: http://69.72.28.81:8501
build-gui-1         | 
```
